pkgs/top-level/all-packages.nix

```diff
@@ -9444,7 +9444,9 @@ with pkgs;
   inherit (callPackage ../servers/plik { })
     plik plikd;
 
-  plex = callPackage ../servers/plex { };
+  plex = callPackage ../servers/plex {
+    buildFHSUserEnv = buildFHSUserEnvBubblewrap;
```

Comment:
@jonringer Do you remember why you did this, instead of just updating the reference in the Plex package itself?